mantidproject/mantid
# Add algorithm to get run information from ISIS journal files

**Description of work.**

This PR adds the algorithm `ISISJournalGetExperimentRuns`, which gets information about runs for a particular experiment from the ISIS journal files.

**Report to:** Max at ISIS

**To test:**

Must be tested on the ISIS network.

Try out the algorithm - see the usage information in its documentation. Check that the results in the output table make sense (you can check them e.g. against JournalViewer or ICat, although note that ICat is sometimes unreliable).

Refs #26897 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
